app/services/hyrax/custom_queries/find_many_by_alternate_ids.rb

```diff
@@ -16,11 +16,15 @@ def initialize(query_service:)
 
       # implements a combination of two Valkyrie queries:
       # => find_many_by_ids & find_by_alternate_identifier
+      #
       # @param alternate_ids [Enumerator<#to_s>] list of ids
-      # @return [Array<Valkyrie::Resource>, Array<ActiveFedora::Base>]
+      # @return [Enumerable<Valkyrie::Resource>, Enumerable<ActiveFedora::Base>]
       def find_many_by_alternate_ids(alternate_ids:)
-        alternate_ids.uniq.map(&:to_s).each_with_object([]) do |id, resources|
-          resources << query_service.find_by_alternate_identifier(alternate_identifier: id)
+        return enum_for(:find_many_by_alternate_ids, alternate_ids: alternate_ids) unless
+          block_given?
```

Comment:
None of the tests use a block with this.  Seems like it should be tested if we are going to support this.

Is the thought that you might want to use the block to process all the returned resources?

Reply:
this is a pretty basic enumerator implementation. `enum_for` uses the block to yield the items, and the tests hit `enum_for`.

all the existing tests hit both this line and the `yield` code below.